### DISCUSSION TITLE
Extension kw don't work

### DISCUSSION THREAD

**Discussion by rsaleev** [Ulauncher/Ulauncher#1267] [Q&A]
![изображение](https://github.com/Ulauncher/Ulauncher/assets/21103312/7f541734-035e-4227-aff2-049b67eeafed)

![изображение](https://github.com/Ulauncher/Ulauncher/assets/21103312/d71a2767-4ee7-496b-a254-bd2dc709b7da)
 
why it's not possible to access extension with `bt`?

**Comment by friday** [comment]
It's not a keyword unless there's a space character after it.
